parquet/src/main/java/org/apache/iceberg/parquet/PruneColumns.java

```diff
@@ -40,7 +43,7 @@ class PruneColumns extends ParquetTypeVisitor<Type> {
   }
 
   @Override
-  public Type message(MessageType message, List<Type> fields) {
+  public Type message(StructType iStruct, MessageType message, List<Type> fields) {
```

Comment:
We generally avoid names like this, I would suggest something like tableStruct

Reply:
Checking other versions, we generally use `expected` for this struct

Reply:
You are right. We do use `expected` elsewhere. Updated.

Reply:
You are right. We do use `expected` elsewhere. Updated.